samples/step-by-step/sample.md

```diff
@@ -7,100 +7,191 @@ redirects:
 - nservicebus/nservicebus-step-by-step-guide
 ---
 
-In this sample shows a very simple ordering system that
+This sample illustrates most of the basic NServiceBus concepts in a step-by-step fashion. Download the code from the link above and run it, or build it  following the instructions here.
+
+
+## Concepts
+
+When complete, this sample will show a very simple ordering system that:
```

Comment:
> When complete, this sample will show a very simple ordering system that:

Rather than start with When complete when the user has hardly begun, how about something like:

The goal of this step-by-step sample is to show how to build a simple system that incorporates using messaging to:
- Send a command from a  client to the server
- Receive a message from a client on the server that publishes a new event 
- Add a new subscriber that can receive published events.